charts/argo-cd/templates/argocd-aggregate-roles.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.rbac.create }}
```

Comment:
I am unsure if we sould use this generic key for this :-)
Normally when setting `rbac.create` on a chart, the chart would not deploy rbac resources at all.

Reply:
I get your point.. i spent quite some time to look up how other charts are dealing with that behaviour. E.g. `argo-rollouts` is using the `clusterInstall` flag for it, `cert-manager` uses `rbac`. I agree that this term is potentially misleading. Do you have a suggestions?

Reply:
IDK what would fit perfectly. But I spottet that we use `createAggregateRoles: true` in argo-workflows: https://github.com/argoproj/argo-helm/blob/master/charts/argo-workflows/values.yaml#L8

Reply:
Sounds good